### PR TITLE
EUI-2138-Section Headings

### PR DESCRIPTION
**Pull request by RiteshHMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-2138

### Change description ###
Change the heading level

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by phillip-whitaker — APPROVED** [review]
Looks like a simple, safe change #famouslastwords

File: package.json [hunk 1]
```diff
@@ -33,7 +33,7 @@
     "ng": "ng",
     "start:ng": "ng serve --port 3000 --host 0.0.0.0 --proxy-config proxy.config.json",
     "build:ng": "ng build",
-    "test:a11y": "webdriver-manager update --versions.chrome 2.40 && ng e2e --webdriver-update=false --protractorConfig=./test/accessibility/config/protractor.conf.js",
+    "test:a11y": "webdriver-manager update --gecko=false --versions.chrome 2.40 && ng e2e --webdriver-update=false --protractorConfig=./test/accessibility/config/protractor.conf.js",
```
Comment (paul-graham) [inline package.json:36]: How many times is that poor gecko going to be denied access?

Comment (paul-graham) [reply package.json:36]: It's discriminatory! 😞

**Review by paul-graham — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (paul-graham) [reply package.json:36]: #PrayForGecko

**Pull request closed without merging**